content/en/news/releases/1.17.x/announcing-1.17.2/index.md

```diff
@@ -67,7 +66,7 @@ Crash when a redirect url without a state parameter is received in the OAuth fil
 
 - **Fixed** an issue causing VMs using auto-registration to ignore labels other than those defined in a `WorkloadGroup`.
   ([Issue #32210](https://github.com/istio/istio/issues/32210))
+  ([PR #44021](https://github.com/istio/istio/pull/44021))
```

Comment:
Point to original PR: https://github.com/istio/istio/pull/44012

Reply:
Done